### PR TITLE
Upgrade to v3.1.9 and improve permissions

### PR DESCRIPTION
**Pull request by tituspijean**
## PR Status

- [x] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by tituspijean** [comment]
!testme

**Comment by tituspijean** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -37,19 +37,22 @@ ram.runtime = "70M"
...
-    type = "user"
+    [install.init_admin_permission]
+    ask.en = "Which group should be able to administer the app and edit flows?"
+    ask.fr = "Quel group devrait pouvoir adminstrer l'app et modifier des flux ?"
+    type = "group"
+    default = "admins"
 
     [install.init_main_permission]
     help.en = "Should the dashboard and HTTP endpoints be publicly accessible?"
     help.fr = "Le tableau de bord et les points d'accès HTTP doivent-ils être publiquement accessibles ?"
```
Comment (alexAubin) [inline manifest.toml:48]:
```suggestion
    help.en = "This corresponds to access to the dashboard and HTTP endpoints"
    help.fr = "Ceci corresponds à l'accès au tableau de bord et aux points d'accès HTTP"
```

(the core already injects the ask string which is the actual question, no need to have "two" questions)

File: manifest.toml [hunk 2]
```diff
@@ -37,19 +37,22 @@ ram.runtime = "70M"
     type = "path"
     default = "/nodered"
 
-    [install.admin]
-    type = "user"
+    [install.init_admin_permission]
+    ask.en = "Which group should be able to administer the app and edit flows?"
+    ask.fr = "Quel group devrait pouvoir adminstrer l'app et modifier des flux ?"
```
Comment (alexAubin) [inline manifest.toml:42]:
```suggestion
    ask.en = "For example, to edit flows"
    ask.fr = "Par exemple, pour modifier les flux"
```

File: tests.toml [hunk 3]
```diff
@@ -8,8 +8,9 @@ test_format = 1.0
...
 # -------------------------------
 
-    [default.test_upgrade_from.3dae97a7]
-    name = "3.0.2~ynh2"
-
-    [default.test_upgrade_from.ca5545af]
-    name = "3.1.4~ynh1 (update nodejs 16 to 20)"
+    [default.test_upgrade_from.6eb3cdb]
+    name = "3.1.8~ynh1"
+    args.domain = "domain.tld"
```
Comment (alexAubin) [inline tests.toml:13]:
```suggestion
    args.domain = "sub.domain.tld"
```

**Comment by alexAubin** [comment]
!testme

File: tests.toml [hunk 4]
```diff
@@ -8,8 +8,9 @@ test_format = 1.0
...
 # -------------------------------
 
-    [default.test_upgrade_from.3dae97a7]
-    name = "3.0.2~ynh2"
-
-    [default.test_upgrade_from.ca5545af]
-    name = "3.1.4~ynh1 (update nodejs 16 to 20)"
+    [default.test_upgrade_from.6eb3cdb]
+    name = "3.1.8~ynh1"
+    args.domain = "sub.domain.tld"
```
Comment (alexAubin) [inline tests.toml:13]:
```suggestion
    args.domain = "sub.domain.tld"
    args.path = "/"
```

**Review by alexAubin — APPROVED** [review]

**Pull request merged**